nifi-commons/nifi-record/src/main/java/org/apache/nifi/serialization/SchemaValidationException.java

```diff
@@ -15,14 +15,16 @@
  * limitations under the License.
  */
 
-package org.apache.nifi.serialization.record;
+package org.apache.nifi.serialization;
 
-public class TypeMismatchException extends RuntimeException {
```

Comment:
i dont think it is ok to change this exeception class name at this juncture and even if it is questionable ok the juice is probably not worth the squeeze.  TypeMismatch and SchemaValidation are pretty much the same thing